### ISSUE TITLE
Add support for jsx-fragments 

### ISSUE THREAD

**Issue by monvillalon** [i18next/i18next-parser#141]
- Is this a bug or a feature request?
Feature Request

- What version are you running (`0.x` or `1.x`)? 
1.0.0-beta30

- Which runner (cli, gulp...)?
cli

## What I'm trying to do?

Extract translations when using jsx fragments shorthand syntax

## How am I doing it?

Using the cli

Here is my config file:

```
{
  locales: ['en', 'de'],
  output: 'locales/$LOCALE/$NAMESPACE.json'
}
```

The command I'm running:

```
i18next -c i18next-parser.config.js
```

## What was the result?

It errors out when parsing a file with fragments with a Acorn's TokenError

## What was I expecting?

To be able to parse the jsx fragments

**Comment by monvillalon** [comment]
I created a PR to fix this issue #140

**Comment by karellm** [comment]
Thanks, I'm closing this in favour of the PR #140, let's keep the conversation there.
